overlord/devicestate/handlers.go

```diff
@@ -199,7 +204,10 @@ func (m *DeviceManager) doGenerateDeviceKey(t *state.Task, _ *tomb.Tomb) error {
 	}
 
 	st.Unlock()
-	keyPair, err := generateRSAKey(keyLength)
+	var keyPair *rsa.PrivateKey
+	timings.Run(perfTimings, "generate-rsa-key", "generating device key pair", func(tm timings.Measurer) {
```

Comment:
Nitpick: I'd slightly prefer `generate-device-key`, but don't feel particularly strongly about that.

Reply:
The principle here is that when measuring around a function, the machine-readable label should try to match the function name. Both the human-readable summary and the surrounding task should make it clear that it is about the device key.